docs/csharp/codedoc.md

```diff
@@ -12,16 +12,464 @@ ms.devlang: dotnet
 ms.assetid: 8e75e317-4a55-45f2-a866-e76124171838
 ---
 
-# 🔧 Documenting your code
-
-> **Note**
-> 
-> This topic hasn’t been written yet! 
->
-> We welcome your input to help shape the scope and approach. You can track the status and provide input on this
-> [issue](https://github.com/dotnet/core-docs/issues/494) at GitHub.
-> 
-> If you would like to review early drafts and outlines of this topic, please leave a note with your contact information in the issue.
->
-> Learn more about how you can contribute on [GitHub](https://github.com/dotnet/core-docs/blob/master/CONTRIBUTING.md).
->
+# XML Documentation
+
+by [Toni Solarin-Sodara](https://github.com/tsolarin)
+
+XML documentation tags are specially formatted comments, added above the definition of a class 
+or its members, used to document your code. You can generate the XML documentation file at compile time 
+by adding `"xmlDoc": true` under `buildOptions` in your `project.json`
+
+* [&lt;summary&gt;](#summary)
+* [&lt;remarks&gt;](#remarks)
+* [&lt;returns&gt;](#returns)
+* [&lt;example&gt;](#example)
+* [&lt;value&gt;](#value)
+* [&lt;exception&gt;](#exception)
+* [&lt;c&gt;](#c)
+* [&lt;code&gt;](#code)
+* [&lt;para&gt;](#para)
+* [&lt;list&gt;](#list)
+* [&lt;see&gt;](#see)
+* [&lt;seealso&gt;](#seealso)
+* [&lt;param&gt;](#param)
+* [&lt;paramref&gt;](#paramref)
+* [&lt;typeparam&gt;](#typeparam)
+* [&lt;typeparamref&gt;](#typeparamref)
+* [&lt;include&gt;](#include)
```

Comment:
I don't think you should write article TOC by hand. Article TOC will be automatically created by the docs-generating tool.